runtime/bajun/src/lib.rs

```diff
@@ -590,6 +594,74 @@ impl pallet_ajuna_awesome_avatars::Config for Runtime {
 	type Randomness = Randomness;
 }
 
+parameter_types! {
+	pub const BasicDeposit: Balance = BAJUN;
+	pub const FieldDeposit: Balance = 100 * MILLI_BAJUN;
+	pub const SubAccountDeposit: Balance = 300 * MILLI_BAJUN;
+	pub const MaxSubAccounts: u32 = 5;
+	pub const MaxAdditionalFields: u32 = 3;
+	pub const MaxRegistrars: u32 = 3;
+}
+
+impl pallet_identity::Config for Runtime {
+	type Event = Event;
+	type Currency = Balances;
+	type BasicDeposit = BasicDeposit;
+	type FieldDeposit = FieldDeposit;
+	type SubAccountDeposit = SubAccountDeposit;
+	type MaxSubAccounts = MaxSubAccounts;
+	type MaxAdditionalFields = MaxAdditionalFields;
+	type MaxRegistrars = MaxRegistrars;
+	type Slashed = Treasury;
+	type ForceOrigin = EnsureRoot<AccountId>;
+	type RegistrarOrigin = EnsureRoot<AccountId>;
+	type WeightInfo = weights::pallet_identity::WeightInfo<Runtime>;
+}
+
+parameter_types! {
+	pub const ProxyDepositBase: Balance = BAJUN;
+	pub const ProxyDepositFactor: Balance = 500 * MILLI_BAJUN;
+	pub const MaxProxies: u32 = 5;
+	pub const MaxPending: u32 = 10;
+	pub const AnnouncementDepositBase: Balance = 200 * MILLI_BAJUN;
+	pub const AnnouncementDepositFactor: Balance = 100 * MILLI_BAJUN;
+}
+
+impl pallet_proxy::Config for Runtime {
+	type Event = Event;
+	type Call = Call;
+	type Currency = Balances;
+	type ProxyType = proxy_type::ProxyType;
+	type ProxyDepositBase = ProxyDepositBase;
+	type ProxyDepositFactor = ProxyDepositFactor;
+	type MaxProxies = MaxProxies;
+	type WeightInfo = weights::pallet_proxy::WeightInfo<Runtime>;
+	type MaxPending = MaxPending;
+	type CallHasher = BlakeTwo256;
+	type AnnouncementDepositBase = AnnouncementDepositBase;
+	type AnnouncementDepositFactor = AnnouncementDepositFactor;
+}
+
+parameter_types! {
+	pub const MaxScheduledPerBlock: u32 = 50;
+	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) * RuntimeBlockWeights::get().max_block;
+	pub const NoPreimagePostponement: Option<u32> = Some(2);
+}
+
+impl pallet_scheduler::Config for Runtime {
+	type Event = Event;
+	type Origin = Origin;
+	type PalletsOrigin = OriginCaller;
+	type Call = Call;
+	type MaximumWeight = MaximumSchedulerWeight;
+	type ScheduleOrigin = EnsureRoot<AccountId>;
+	type OriginPrivilegeCmp = frame_support::traits::EqualPrivilegeOnly;
+	type MaxScheduledPerBlock = MaxScheduledPerBlock;
+	type WeightInfo = ();
+	type PreimageProvider = ();
```

Comment:
Could you also add `pallet_preimage` and use it for PreimageProvider?
```suggestion
	type PreimageProvider = Preimage;
```
I'm not sure why these pallets are decoupled :smile:

Reply:
Yeah, strange decision